src/OnlineSales/Migrations/20230713001357_AddPublishedAtToContent.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace OnlineSales.Migrations
+{
+    /// <inheritdoc />
+    public partial class AddPublishedAtToContent : Migration
+    {
+        /// <inheritdoc />
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.AddColumn<DateTime>(
```

Comment:
@gsplakmal we need default value for PublishedAt to be set to the current CreatedAt value for all existing records. Please add mentioned logic to the migration as well.

Reply:
updated with sql script to set default data.